### DISCUSSION TITLE
Courseplay and pausing jobs

### DISCUSSION THREAD

**Discussion by Patrkb512** [Courseplay/Courseplay_FS22#2923] [Q&A]
Not sure if this has been discussed before ( I did a couple of searches), but what is the best way to 'pause' a job while you untangle tractors or manually fix a dodgy unload?  

 I know I can restart a job with 'nearest waypoint', but courses often overlap in the kind of area that we would be most likely to need manual intervention. I would love to be able to 'pause' a job, assume manual control over the tractor, do what needs to be done, and have CP 'remember' the last waypoint then automatically resume the course at the waypoint that was last passed. 

That, or a feature to let us specify a waypoint on start such as '399/1200' sort of option.  This would open up both a 'resume' option, and an option to have 2 tools start in different places on the course.

**Comment by Tensuko** [comment]
Just use start on latest waypoint and not nearest.
A pause option wont happen.
